java/lib/src/main/java/io/amplica/graphsdk/Graph.java

```diff
@@ -58,16 +57,19 @@ public void applyActions(Actions actions) throws BaseGraphSdkException {
         }
     }
 
-    public List<Updates.Update> forceRecalculateGraph(long dsnpUserId) throws BaseGraphSdkException, InvalidProtocolBufferException {
+    public List<Updates.Update> forceRecalculateGraph(long dsnpUserId)
+            throws BaseGraphSdkException, InvalidProtocolBufferException {
         try (NativeHandleGuard guard = new NativeHandleGuard(this)) {
             var raw = Native.forceCalculateGraphs(guard.nativeHandle(), dsnpUserId);
             return Updates.parseFrom(raw).getUpdateList();
         }
     }
 
-    public List<DsnpGraphEdges.DsnpGraphEdge> getConnections(long dsnpUserId, ConnectionType connectionType, boolean includePending) throws BaseGraphSdkException, InvalidProtocolBufferException {
+    public List<DsnpGraphEdges.DsnpGraphEdge> getConnections(long dsnpUserId, ConnectionType connectionType,
```

Comment:
Please don't forget to add that new api that exports updates only for a certain user to JVM